extensions/broker-server/src/test/java/de/sovity/edc/ext/brokerserver/services/api/AddConnectorsApiTest.java

```diff
@@ -56,27 +58,100 @@ void testAddConnectors() {
             client.brokerServerApi().addConnectors(ADMIN_API_KEY, List.of());
 
             client.brokerServerApi().addConnectors(ADMIN_API_KEY, Arrays.asList(
+                null,
+                "",
+                "  ",
+                "\t",
+                "http://a",
+                "http://b"
+            ));
+
+            assertThat(client.brokerServerApi().connectorPage(new ConnectorPageQuery()).getConnectors())
+                .extracting(ConnectorListEntry::getEndpoint)
+                .containsExactlyInAnyOrder("http://a", "http://b");
+
+            client.brokerServerApi().addConnectors(ADMIN_API_KEY, Arrays.asList(
+                "http://b",
+                " http://b\r\n",
+                "http://c"
+            ));
+
+            assertThat(client.brokerServerApi().connectorPage(new ConnectorPageQuery()).getConnectors())
+                .extracting(ConnectorListEntry::getEndpoint)
+                .containsExactlyInAnyOrder("http://a", "http://b", "http://c");
+        });
+    }
+
+    @Test
+    void testAddConnectorsWithMdsIds() {
+        TEST_DATABASE.testTransaction(dsl -> {
+            client.brokerServerApi().addConnectorsWithMdsIds(ADMIN_API_KEY, new ConnectorCreationRequest(List.of()));
+
+            client.brokerServerApi().addConnectorsWithMdsIds(ADMIN_API_KEY, new ConnectorCreationRequest(Arrays.asList(
+                new AddedConnector(
                     null,
+                    "MDSL1234"
+                ),
+                new AddedConnector(
                     "",
+                    "MDSL1234"
+                ),
+                new AddedConnector(
                     "  ",
+                    "MDSL1234"
+                ),
+                new AddedConnector(
                     "\t",
+                    "MDSL1234"
+                ),
+                new AddedConnector(
                     "http://a",
-                    "http://b"
-            ));
+                    "MDSL1234"
+                ),
+                new AddedConnector(
+                    "http://b",
+                    "MDSL1234"
+                ),
```

Comment:
comment: we could add some whitespace to the beginning/end of one of the two's URLs here to test that case